projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.ts

```diff
@@ -0,0 +1,128 @@
+import {ChangeDetectionStrategy, Component, OnInit, Input, Output, EventEmitter, ChangeDetectorRef} from '@angular/core';
+import {RasterSymbology} from '../../layers/symbology/symbology.model';
+import {Color} from '../color';
+import {ColorAttributeInput} from '../color-attribute-input/color-attribute-input.component';
+import {ColorBreakpoint} from '../color-breakpoint.model';
+import {PaletteColorizer} from '../colorizer.model';
+
+@Component({
+    selector: 'geoengine-color-palette-editor',
+    templateUrl: './color-palette-editor.component.html',
+    styleUrls: ['./color-palette-editor.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ColorPaletteEditorComponent implements OnInit {
+    // Output and Input for communication with the parent "raster-symbology-editor" component
```

Comment:
Please don't hint a possible parent component but state what these inputs and outputs are.

Reply:
Fixed the comment in [959e1fe](https://github.com/geo-engine/geoengine-ui/pull/614/commits/959e1fec8e9dbc3c99e72add43ed6d0bd294de48)